import exceptions

# <exceptions>: call function(s) by passing argument(s)
# exceptions.string_exceptions()
# exceptions.handle_list_exceptions()
exceptions.handle_tuple_exceptions()

'''print("Please enter any two integer values")
a = int(input())
b = int(input())
exceptions.handle_runtime_exceptions(input(), input())
'''